map-matching/api-reference.md

```diff
@@ -21,13 +21,67 @@ The map-matching service is in active development. You can follow the [Mapzen bl
 The default logic for the OpenStreetMap tags, keys, and values used when routing are documented on an [OSM wiki page](http://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Valhalla).
 
 
+## Improving Map-Matching Results
+
```

Comment:
Maybe add something like `Criteria for accurate map-match results`
before the list